Bunch of updates since v1.1

strangerstudios opened the pull request.

ideadude committed:
Added strangerstudios to readme

ideadude committed:
* Admin accounts (specifically users with "manage_options" capability) are excluded from bounces. This will eventually be a setting once we setup a settings page.
* Readme changes.

Commit:
Corrected a typo

Replaced $_SERVER['REMOVE_ADDR'] with $_SERVER['REMOTE_ADDR']

ideadude committed:
Merge pull request #1 from mnakalay/master

Corrected a typo

ideadude committed:
Fixed typo in variable used to create the session ID. (Thanks, chuckingit on GitHub.)

ideadude committed:
* Fixed some typos in the variables used to generate the session ids.
* The fakesessid_{user_login} transients are now storing arrays of session ids. This allowed for multiple (but limited) sessions per user if wanted.
* Added wp_bouncer_ignore_admins filter, if returning false even admins will be bounced.
* Added wp_bouncer_redirect_url filter, which can be used to change the URL redirected to after being bounced.
* Added wp_bouncer_number_simultaneous_logins filter, which can be set to limit logins to a number other than 1. 0 means unlimited logins.
* Added wp_bouncer_login_flag in case you want to hook in and do something right before bouncing (or potentially stop the bouncing).

ideadude committed:
1.3 updates

eighty20results committed:
FIX: Typo in variable name

ideadude committed:
Tested up

ideadude committed:
Merge pull request #2 from eighty20results/patch-1

FIX: Typo in variable name

ideadude committed:
Readme update, version to 1.3.1

ideadude committed:
updating readme with all hooks and filters; better description and information

lkraav commented:
Heya all. Is this PR going anywhere any time soon? Or was a better plugin discovered in the meanwhile?

zackdn committed:
Update wp-bouncer.php

get_transient was being incorrectly called and causing the transient not to be created and thus breaking the entire plugin.

zackdn committed:
Filter modification

I changed this to allow add_filter within my functions.php file within my child theme.

zackdn committed:
Update wp-bouncer.php

ideadude committed:
Merge pull request #3 from zackdn/patch-1

Update wp-bouncer.php

ideadude committed:
Turned get_redirect_url into a method. Some WPCS stuff.

ideadude committed:
Better description in doc block

ideadude committed:
Code cleanup

ideadude committed:
Tweaking plugin header, author, urls.

ideadude committed:
Added JS checks

ideadude committed:
Make sure to return data even when not flagged or the JS will error out.

ideadude committed:
Linux line endings

ideadude committed:
Version and readme update.

ideadude committed:
Adding WP_BOUNCER_HEARTBEAT_CHECK info to readme.

ideadude committed:
Linux line endings.

ideadude committed:
Readme heading

kimcoleman committed:
Updating tested up to value

ideadude committed:
BUG FIX: Fixed issue where users were not redirected to the warning page when logged out.

andrewlimaza committed:
BUG FIX: Adjusted metatags and HTTPS URLS

BUG FIX: Adjusted URLs to be https and adjusted meta tags to be be noindex/nofollow.

Note: noodp,noydir isn't supported anymore.

andrewlimaza committed:
Update bounced functionality.

Removed login-warning.php, no longer needed.

ENHANCEMENT: when user is bounced, redirect to login screen with an error message.

ideadude committed:
Merge pull request #7 from andrewlimaza/dev

BUG FIX: Adjusted metatags and HTTPS URLS

ideadude committed:
Merge branch 'redirect-fix' of https://github.com/andrewlimaza/wp-bouncer into andrewlimaza-redirect-fix

Conflicts:
	login-warning.php

ideadude committed:
we're using a different version number

ideadude committed:
Removing this file

ideadude committed:
Merge branch 'andrewlimaza-redirect-fix' into dev

kimcoleman committed:
Update readme.txt

ideadude committed:
tested on 6.1. all good.

kimcoleman committed:
Adding repository files, updating readme for .org

kimcoleman committed:
Merge pull request #10 from kimcoleman/dev

Adding repository files, updating readme for .org

kimcoleman committed:
Fixing misspelled constant for version

kimcoleman committed:
Updating readme changelog

kimcoleman committed:
Localization; Added filter for ajax timeout

kimcoleman committed:
Updating readme

kimcoleman committed:
Merge pull request #11 from kimcoleman/ajax-timeout-filter

Ajax timeout filter; Localization

github-actions[bot] committed:
🔄 Regenerate translation files
